bundles/org.openhab.binding.teslapowerwall/README.md

```diff
@@ -0,0 +1,99 @@
+# TeslaPowerwall Binding
+
+This binding enables the capture of key data from a Tesla Powerwall 2 into openHAB.
+
+## Supported Things
+
+- `powerwall` Tesla Powerwall 2
+
+## Discovery
+
+The binding does not support auto discovery.
+
+## Thing Configuration
+
+As a minimum, the hostname is needed:
+
+* hostname - The hostname of the Tesla Powerwall 2. Defaults to powerwall to avoid SSL certificate issues
+* email - the email of the local account on the Powerwall that the installer provided
+* password - the password of the local account on the Powerwall that the installer provided
+* refresh - The frequency with which to refresh information from the Tesla Powerwall2 specified in seconds. Defaults to 10 seconds.
+
+## Channels
+
+| channel id             | type           | description                                                                           |
+|------------------------|----------------|---------------------------------------------------------------------------------------|
+| gridstatus             | String         | Current status of the Power Grid
+| batterysoe             | Number:Percent | Current battery state of charge
```

Comment:
```suggestion
| battery-soe             | Number:Dimensionless | Current battery state of charge
``` 

See other comment for naming convention, this comment  is mainly about the type. Number:Percent does not exist. 
For reference: https://www.openhab.org/docs/concepts/units-of-measurement.html#list-of-units

Also update the thing-type.xml accordingly

Reply:
Done